models/safe/zkevm/safe_zkevm_schema.yml

```diff
@@ -0,0 +1,158 @@
+version: 2
+
+models:
+  - name: safe_zkevm_safes
+    meta:
+      blockchain: zkevm
+      project: safe
+      contributors: danielpartida
+    freshness:
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
+    config:
+      tags: ['safe', 'zkevm']
+    description: "Safe addresses"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - address
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "The blockchain on which the Safe is deployed"
+      - &address
+        name: address
+        description: "Safe contract address"
+      - &creation_version
+        name: creation_version
+        description: "Version of initially created safe"
+      - &block_date
+        name: block_date
+      - &block_month
+        name: block_month
+      - &creation_time
+        name: creation_time
+        description: "Date/time of safe creation"
+      - &tx_hash
+        name: tx_hash
+
+  - name: safe_zkevm_matic_transfers
+    meta:
+      blockchain: zkevm
+      project: safe
+      contributors: danielpartida
+    freshness:
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
+    config:
+      tags: ['safe', 'transfers', 'zkevm']
+    description: "MATIC transfers for safes"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - tx_hash
+            - trace_address
+            - amount_raw
+    columns:
+      - *blockchain
+      - &symbol
+        name: symbol
+        description: "Symbol of native gas token: MATIC"
+      - *address
+      - *block_date
+      - *block_month
+      - &block_time
+        name: block_time
+        description: "Date of MATIC transfer"
+      - &amount_raw
+        name: amount_raw
+        description: "Raw amount of transferred MATIC"
+      - *tx_hash
+      - &trace_address
+        name: trace_address
+      - &amount_usd
+        name: amount_usd
+        description: "USD amount of transferred MATIC"
+
+  - name: safe_zkevm_singletons
+    meta:
+      blockchain: zkevm
+      project: safe
+      contributors: danielpartida
+    freshness:
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
+    config:
+      tags: ['safe', 'singletons', 'zkevm']
+    description: "Singletons addresses used with Safes"
+    columns:
+      - name: address
+        description: "Safe contract address"
+        tests:
+          - unique
+          - not_null
+
+  - name: safe_zkevm_transactions
```

Comment:
this one still needs the unique combo test on the keys [here](https://github.com/duneanalytics/spellbook/pull/5758/files#diff-12305aeb69517b8ef512cd29aa43cc6daf46d4c9d5e1b14e78b1b51b13f2548eR7)